enacts/config-enacts.yaml

```diff
@@ -0,0 +1,72 @@
+logo: anacimLogo.jpg
```

Comment:
See https://github.com/iridl/python-maprooms/pull/481#discussion_r1984118167 regarding the name of this file.